dipy/reconst/fwdti.py

```diff
@@ -239,7 +239,7 @@ def predict(self, gtab, S0=1):
 def wls_iter(
     design_matrix, sig, S0, Diso=3e-3, mdreg=2.7e-3, min_signal=1.0e-6, piterations=3
 ):
-    """Applies weighted linear least squares fit of the water free elimination
+    r"""Applies weighted linear least squares fit of the water free elimination
```

Comment:
same here

Reply:
My bad. I may have added it thinking that a warning in this method was raised by the math expressions. Removed in 1b8fea5.